Twig/Extension/AceEditorExtension.php

```diff
@@ -81,7 +81,7 @@ public function includeAceEditor()
 
         if (!$this->ckeditorIncluded) {
             $jsPath = $this->environment
-                ->getExtension('assets')
+                ->getExtension('asset')
```

Comment:
you need to check first if the asset extension is aviable, otherwise you take the assets extension, you don't have to loose backward compability